src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -1,24 +1,24 @@
 //ie11 fixes
-:host-context([theme="dark"]) {
+:host([theme="dark"]) {
```

Comment:
you can just delete all these. If you look up at `calcite-accordion.scss` you'll see that these styles are declared in the main host for the component. 

When you see `:host-context` selectors they are generally duplicating existing styles and can be fully deleted (this will remove support for any of those features from ie11)